CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-## [UNRELEASED] neptune 1.9.2
+## [UNRELEASED] neptune 1.10.0
+
+### Features
+- Added auto-scaling for image logging ([#1664](https://github.com/neptune-ai/neptune-client/pull/1664))
```

Comment:
We should probably mention that it's pixel range auto-scaling as scaling wrt. images may mean changing size.